hyp3_autorift/process.py

```diff
@@ -32,8 +32,7 @@
 gdal.UseExceptions()
 
 S3_CLIENT = boto3.client('s3')
-S2_SEARCH_URL = 'https://earth-search.aws.element84.com/v0/collections/sentinel-s2-l1c/items'
-S2_WEST_BUCKET = 's2-l1c-us-west-2'
+S2_GRANULE_DIR = 'https://storage.googleapis.com/gcp-public-data-sentinel-2/tiles/'
```

Comment:
The `/` at the end here is leading to a double `//` in the manifest safe url generated on line 123, e.g.

`https://storage.googleapis.com/gcp-public-data-sentinel-2/tiles//38/E/MQ/S2B_MSIL1C_20200419T060719_N0209_R105_T38EMQ_20200419T091056.SAFE/manifest.safe` (generates an error)
`https://storage.googleapis.com/gcp-public-data-sentinel-2/tiles/38/E/MQ/S2B_MSIL1C_20200419T060719_N0209_R105_T38EMQ_20200419T091056.SAFE/manifest.safe` (successfully downloads the file)

I think dropping the trailing `/` is the easiest path forward? Or use something fancier like `os.path.join` or `urllib.parse.urljoin` to make it not matter whether there's an extra `/` or not?

```suggestion
S2_GRANULE_DIR = 'https://storage.googleapis.com/gcp-public-data-sentinel-2/tiles'
```